include/odp/api/spec/time.h

```diff
@@ -76,6 +77,24 @@ odp_time_t odp_time_local(void);
  */
 odp_time_t odp_time_global(void);
 
+/**
+ * Current local time in nanoseconds
+ *
+ * Like odp_time_local(), but time stamp value is converted to nanoseconds.
+ *
+ * @return Local time stamp in nanoseconds
+ */
+uint64_t odp_time_local_ns(void);
+
+/**
+ * Current global time in nanoseconds
+ *
+ * Like odp_time_global(), but time stamp value is converted to nanoseconds.
```

Comment:
Perhaps  `..., but the time stamp value is converted into nanoseconds`. Same for `odp_time_local_ns ()`.

Reply:
fixed in v3